istioctl/cmd/root.go

```diff
@@ -184,6 +185,7 @@ debug and diagnose their Istio mesh.
 
 	rootCmd.AddCommand(experimentalCmd)
 	rootCmd.AddCommand(proxyconfig.ProxyConfig(ctx))
+	rootCmd.AddCommand(ztunnelconfig.ZtunnelConfig(ctx))
```

Comment:
I think we should add this to the experimental command first and promote it if needed.

Reply:
@hanxiaop @howardjohn @bleggett 

PR for that update: https://github.com/istio/istio/pull/50378